app/modules/linshare.receivedShare/receivedShareController.js

```diff
@@ -127,19 +122,6 @@
         });
       }
 
-      /**
-       * @name addSelectedDocument
-       * @desc add contacts to list of new contacts to create
-       * @param {Object} document - document to add to the list of selected documents
-       * @memberOf LinShare.receivedShare.ReceivedController
-       */
-      function addSelectedDocument(document) {
```

Comment:
this is not used anymore?

Reply:
yes, replaced it with `tableParamsService.toggleItemSelection`